src/qililab/qprogram/structured_program.py

```diff
@@ -250,10 +262,22 @@ def _float_variable(label: str, domain: Domain = Domain.Scalar) -> FloatVariable
 
         if domain == Domain.Time:
             return _int_variable(label, domain)
-        if domain in [Domain.Frequency, Domain.Phase, Domain.Voltage]:
+        if domain in [Domain.Frequency, Domain.Phase, Domain.Voltage, Domain.Flux]:
             return _float_variable(label, domain)
         raise NotImplementedError
 
+    def crosstalk(self, crosstalk: CrosstalkMatrix | None = None, calibration: Calibration | None = None):
```

Comment:
And this is better named `set_crosstalk`.

```suggestion
    def set_crosstalk(self, crosstalk: CrosstalkMatrix | None = None, calibration: Calibration | None = None):
```